src/helpers/url.js

```diff
@@ -33,6 +33,9 @@ const getSearchParam = (name) => {
 }
 
 const getWopiUrl = ({ fileId, title, readOnly, closeButton, revisionHistory }) => {
+	// Only set the revision history parameter if the versions app is enabled
+	revisionHistory = revisionHistory && window?.oc_appswebroots?.files_versions
+
```

Comment:
Is this a reliable way to check if an app is enabled? I mean are we sure it won't get removed anytime soon?

If not we could get the information via initial state I guess.

Reply:
Yeah, ideally we should have initial state for that, but I'd stick to this for now to keep it back portable and maybe pass over more stuff through initial state when getting back to #2258

Reply:
:+1: